### DISCUSSION TITLE
Error running examples 8 through 12

### DISCUSSION THREAD

**Discussion by 13ryc** [NatLabRockies/floris#409] [Q&A] [answered]
Hi, after updating FLORIS to the most recent version, I have been trying to play around with the new example files. When trying to run examples 8,9,10,11 and 12 I have recieved the following error:

ModuleNotFoundError: No module named 'floris.tools.optimization.yaw_optimization'

Since the error is associated with importing the module I tried to ammend the issue myself. When doing so, I noticed that the __init__.py file within the yaw_optimization folder (floris\tools\optimization\yaw_optimization\__init__.py) was blank. I tried filling it in with the following code:

from . import (
    yaw_optimization_base,
    yaw_optimization_tools,
    yaw_optimizer_scipy,
    yaw_optimizer_sr,
)

This however did not solve the problem. So my question is, why is the module not found and is there an easy way to fix this issue and get these examples to run?

Kind regards

**Comment by rafmudaf** [comment, marked as answer]
Hi @13ryc This was fixed in #374 and it is in the `main` branch but it has not yet made it to a release. You could either clone the repository and install it or wait until the next release. We are anticipating finalizing v3.1 (see #405) in the next day or so.
